tests/http/src/test/java/org/mule/runtime/module/http/functional/listener/HttpListenerMethodRoutingTestCase.java

```diff
@@ -65,7 +66,7 @@ public void callWithMethod() throws Exception
     @Test
     public void callWithMethodEmptyBody() throws Exception
     {
-        sendRequestAndAssertMethod(null);
+        sendRequestAndAssertMethod("");
```

Comment:
The called method receives String, and sending an empty one is okay for this test.